src/MolSim.cpp

```diff
@@ -17,28 +20,37 @@
 int main(const int argc, const char* argv[]) {
     namespace fs = std::filesystem;
 
-    std::cout << "Started " << argv[0] << std::endl;
-
     // Initialize the simulation environment, readers and writers.
     Environment env(argc, argv);
 
-    ParticleContainer container;
+    spdlog::info("Started {}", argv[0]);
+
+    // Initialize the calculator
+    std::unique_ptr<physicsCalculator::Calculator> calculator { nullptr };
 
-    std::unique_ptr<inputReader::Reader> fileReader { new inputReader::FileReader() };
-    fileReader->readFile(container.get_particles(), env.get_input_file_name());
+    switch (env.get_calculator_type()) {
+    case GRAVITY:
+        calculator.reset(new physicsCalculator::GravityCalculator(env));
+        break;
 
-    // Comment out befor final merge (This code removes all old .vtu and .xyz files directly within the build folder)
-    // std::cout << "Cleaning up old output files!" << std::endl;
-    // fs::path dir_path { "." };
-    // for (auto const& dir_entry : fs::directory_iterator { dir_path }) {
-    //     if (dir_entry.path().extension() == ".vtu" || dir_entry.path().extension() == ".xyz") {
-    //         fs::remove(dir_entry.path());
-    //     }
-    // }
```

Comment:
dead code, remove if not being used. create a separate branch if it is under development.